sound/soc/sof/sof-priv.h

```diff
@@ -167,6 +167,30 @@ struct snd_sof_dsp_ops {
 	snd_pcm_uframes_t (*pcm_pointer)(struct snd_sof_dev *sdev,
 					 struct snd_pcm_substream *substream); /* optional */
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG_PROBES)
+	/* Except for probe_pointer, all probe ops are mandatory */
```

Comment:
I don't get what you are trying to say with 'except for probe pointer'? there is no probe pointer below

Reply:
There is move few lines below. Am I missing something here?

Reply:
I don't get the answer "There is move few lines below", is there a typo with 'move'?

Reply:
Yeah, sorry for the typo.

Reply:
so what did you mean?